### PR TITLE
disable measures ordering on fontend

### PR DESCRIPTION
**Pull request by alexey-klimuk**
Before
screenshot from 2017-10-02 17-19-41

After
screenshot from 2017-10-02 17-18-58

### CODE REVIEW

**Comment by alexey-klimuk** [comment]
[[TARIFF17] Measure ordering](https://trello.com/c/r8vBlBIZ/1943-tariff17-measure-ordering)

**Comment by rusllonrails** [comment]
Looks ok :+1:

File: app/models/measure.rb [hunk 1]
```diff
@@ -73,6 +73,6 @@ def additional_code_sort
   end
 
   def key
```
Comment (matthewford) [inline app/models/measure.rb]: we can probably remove the method altogether right

Comment (alexey-klimuk) [reply app/models/measure.rb]: yeah, can be removed

**Pull request merged**